cime_config/cesm/machines/config_machines.xml

```diff
@@ -1978,7 +1978,7 @@
     </environment_variables>
   </machine>
   <default_run_suffix>
-    <default_run_exe>$config{'EXEROOT'}/cesm.exe </default_run_exe>
+    <default_run_exe>$config{EXEROOT}/$config{EXEC_SE}</default_run_exe>
```

Comment:
Make this an xml variable so that tests can build multiple executables with different names.